src/components/Article.jsx

```diff
@@ -1 +1,32 @@
 // export const Article = () => ();
+import React from 'react';
+
+function dateISO(date) {
+  return date.toISOString().slice(0, 10);
+}
+
+function dateLocaleString(date) {
+  return date.toLocaleDateString('en-US', {
+    year: 'numeric',
+    month: 'long',
+    day: 'numeric',
+  });
+}
```

Comment:
ok. But you can do this inside the functional component before returning the result. assign result of actions to the variables and use them)